README.md

```diff
@@ -5,6 +5,10 @@ We are in the formative steps of building out infrastructure, process and collab
 ## What does the Node.js End User Feedback team do?
 The Node.js End User Feedback team works with Node.js end users to provide a bi-directional feedback channel for the Node.js project through surveys, meetings and outreach.
 
+### Initiatives
```

Comment:
Given that this is for the the User Feedback Initiative, I'd suggest updating this heading to avoid confusion around an initiative with initiatives 😅

Reply:
"Projects"? I'm open to suggestions for what a better word might be.

Reply:
Projects works 👍